playbooks/terraform-provider-openstack-acceptance-test-public-clouds/post.yaml

```diff
@@ -0,0 +1,61 @@
+- hosts: all
+  become: yes
+  roles:
+    - export-cloud-openrc
+  tasks:
+    - name: Clean up resources for public clouds & terraform-provider-openstack acceptance tests
+      shell:
+        cmd: |
+          set -e
+          set -x
+          # clean up instances
+          for ins_id in `openstack server list -f value -c ID -c Name |grep instance_1 | awk -F ' ' '{print $1}'`
```

Comment:
you can add option `--no-name-lookup` for command `server list`, that is speeder than default option